.gitignore

```diff
@@ -0,0 +1 @@
+/docs/_build/
```

Comment:
Please remove everything under _build and add the following to .gitignore:
docs/_build/
docs/venv/
*.swp
*.pyc
docs/bin/
docs/include/
docs/lib/
docs/lib64
docs/pip-selfcheck.json
docs/.Python